services/resource-getter.js

```diff
@@ -17,18 +16,6 @@ function ResourceGetter(model, params) {
             ' does not exist.');
         }
 
-        // NOTICE: Do not use "toJSON" method to prevent issues on models that
-        //         override this method.
-        record = record.get({ plain: true });
-
-        // Ensure the Serializer set the relationship links on has many
-        // relationships by setting them to an empty array.
-        _.values(model.associations).forEach(function (association) {
-          if (['HasMany', 'BelongsToMany'].indexOf(association.associationType) > -1) {
-            record[association.associationAccessor] = [];
-          }
-        });
```

Comment:
Useless code (2 years old), it seems that the serializer does not need this anymore